stdlib/public/stubs/Stubs.cpp

```diff
@@ -445,11 +445,27 @@ __mulodi4(di_int a, di_int b, int* overflow)
 #endif
 
 #if defined(__CYGWIN__) || defined(_WIN32)
+  #define strncasecmp _strnicmp
+#endif
+
+static bool _swift_stdib_strissnan(const char *nptr) {
```

Comment:
Perhaps, to help legibility, and since this isn't a shim for some libc function, we could give it a more readable name in the same vein as `swift_floatingPointToString`, such as `swift_stringIsSignalingNaN`?